### ISSUE TITLE
Improve amp-carousel performance

### ISSUE THREAD

**Issue by ericlindley-g** [ampproject/amphtml#2373]
Hoping to figure out how to improve amp-carousel performance, as reports have come in detailing some jankiness:

• Delay between tap/swipe and slide change
• Stuttering images on swipe
• Slow loading images

**Comment by ericlindley-g** [comment]
Have repro'd on iPhone 6, Nexus 5x, and Moto X

**Comment by cramforce** [comment]
@ericlindley-g Have a good sample doc?

**Comment by rudygalfi** [comment]
Check out https://www.google.com/amp/s/www.washingtonpost.com/amphtml/news/monkey-cage/wp/2016/03/01/what-is-the-chance-that-trump-dominates-super-tuesday-but-doesnt-win-the-nomination/#fpstate=amp_viewer

**Comment by ericlindley-g** [comment]
The Eng team has done some performance work recently, and I'm unable to reproduce the jank I was seeing earlier. Well done!  (there are some additional UX changes to consider that could lessen perceived jank, as well, like heightening sensitivity for the swipe gesture, but I'll follow up on that in another issue)

**Comment by aghassemi** [comment]
It's time for this again :)  reopening.

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @cathyxz Do you have any updates?

**Comment by aghassemi** [comment]
carousel v2 handles this.
